src/Cell.cpp

```diff
@@ -1365,7 +1365,7 @@ void Cell::ringify(std::vector<Cell*>& subcells, double max_radius) {
                    (*iter3)->getId());
 
         /* Create a new Cell clone */
-        Cell* ring = (*iter3)->clone();
+        Cell* ring = (*iter3)->clone();//need false option?
```

Comment:
false option should not be used here. Otherwise we will delete the sector surfaces we just added in sectorize

Reply:
Agree!